.github/workflows/buildRelease.yml

```diff
@@ -13,17 +13,21 @@ jobs:
         uses: actions/checkout@v4
         with:
           submodules: recursive
-      - name: Build Release
+      - name: Build Release Artifacts
         env:
           TAG: ${{ github.ref_name }}
         run: |
-          echo "Perform and build steps here such as docs, .sch, .xsl, zip files, etc..."
+            VERSION=$(echo $TAG | sed -nre 's/^[^0-9]*(([0-9]+\.)*[0-9]+).*/\1/p')
+            echo "Build OVAL Version ${VERSION}"
+            make OVAL_VERSION=${VERSION}
       - name: Attach Release Artifacts
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           OWNER: ${{ github.repository_owner }}
           REPO: ${{ github.event.repository.name }}
           TAG: ${{ github.ref_name }}
         run: |
-            SCHEMA_FILES=$(ls oval-schemas/*.xsd)
-            gh release upload $TAG $SCHEMA_FILES --clobber --repo $OWNER/$REPO
+            # SCHEMA_FILES=$(ls oval-schemas/*.xsd)
+            # gh release upload $TAG $SCHEMA_FILES --clobber --repo $OWNER/$REPO
+            VERSION=$(echo $TAG | sed -nre 's/^[^0-9]*(([0-9]+\.)*[0-9]+).*/\1/p')
+            gh release upload $TAG schemas-${VERSION}.zip oval_docs-${VERSION}.zip --clobber --repo $OWNER/$REPO
```

Comment:
This requires an existing release. I'd suggest you use a release create instead. Even if you don't want to automatically release, you can set the command to create a draft release:

`gh release create $TAG "$DOC_DIR".zip -d --generate-notes --verify-tag --repo $OWNER/$REPO`

Reply:
There is also the option to verify if a release already exists, then just update the assets.
Something like:
```
if $(gh release list | head -n2 | cut -d$'\t' -f3 | grep -w -q "$TAG"); then 
    gh release upload $TAG schemas-${VERSION}.zip oval_docs-${VERSION}.zip --clobber --repo $OWNER/$REPO
else
    gh release create $TAG schemas-${VERSION}.zip oval_docs-${VERSION}.zip -d --generate-notes --verify-tag --repo $OWNER/$REPO
fi
```

Reply:
@DeafBDoor right now the triggers for this workflow is such that the release already exists, so unless a manual trigger is added, the `gh release create` stuff isn't needed.  Not sure if there is really a need for a manual trigger for this workflow, drafting a release with the `/releases/new` URL and unchecking the "Set as a pre-release" and "Set as the latest release" boxes, and clicking publish.  Saving the release as a draft doesn't trigger the workflow, but I don't think it should, as I see that state as where one edits the description and release notes for a release (the auto-generated and attached source files also don't get attached in that state).

If people think there is a need/use case to have a manual trigger for this workflow, I can add it.